subprojects/groovy-json/src/main/java/groovy/json/JsonOutput.java

```diff
@@ -59,9 +59,8 @@
     private static final char[] EMPTY_STRING_CHARS = Chr.array(QUOTE, QUOTE);
 
     private static final String NULL_VALUE = "null";
-    private static final String JSON_DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ssZ";
+    private static final String JSON_DATE_FORMAT = "yyyy-MM-dd'T'HH:mm:ssXX";
```

Comment:
This will throw an `java.lang.IllegalArgumentException` in Java 6 and according to http://www.groovy-lang.org/install.html#_prerequisites JDK 6+ works with Groovy 2.4.